### ISSUE TITLE
Apollo Angular import treated as default instead of namespaced

### ISSUE THREAD

**Issue by NickClark** [dotansimha/graphql-code-generator#1504]
**Describe the bug**
The generated code currently imports Apollo Angular as a default import instead of a namespaced import. Maybe there's a way in Webpack to make it generate a default export when none is provided by the library, but I'm unaware of how to accomplish that.

**To Reproduce**
Just add the following into any valid project:

```
    plugins:
      - typescript
      - typescript-operations
      - typescript-apollo-angular
```

and you'll get `import Apollo from "apollo-angular;"

**Expected behavior**
You should get `import * as Apollo from "apollo-angular;"

**Environment:**

- OS: macOS 10.13.6
- Codegen: 1.0.0
- Node: 10

**Comment by dotansimha** [comment]
Fixed in 1.0.1 🚀

**Comment by NickClark** [comment]
Thanks! 🎊
